boards/shields/xenvm_dom0/boards/qemu_cortex_a53.overlay

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &sram0;
+/delete-node/ &pcie;
+/delete-node/ &uart0;
+
+/ {
+	chosen {
+		/delete-property/ zephyr,flash;
+		/delete-property/ zephyr,pcie-controller;
+	};
+
+	/*
+	 * This node may differs on different setups, please check
+	 * following line in Xen boot log to set it right:
+	 * (XEN) Grant table range: 0x00000040200000-0x00000040240000
```

Comment:
Please expand this comment by describing that Xen passes actual gntab ranges in device tree, but Zephyr is not capable of parsing device tree in run-time.

Reply:
Fixed